pkg/elemental/models/elemental-resource.js

```diff
@@ -5,4 +5,8 @@ export default class ElementalResource extends SteveModel {
   get listLocation() {
     return createElementalRoute('resource', { resource: this.type });
   }
+
+  get doneRoute() {
```

Comment:
In epinio i also had to override `parentLocationOverride`. From what i remember that handles the resource link in the header of detail/edit resource pages. Worth double checking that you need/don't need that

Reply:
@richard-cox just updated the code with https://github.com/rancher/elemental-ui/pull/60/commits/212c54833791cd8549d6d1444c694e0a1d90b8cf. Thanks for the feedback! 🙇